mythril_core/src/logger.rs

```diff
@@ -1,22 +1,29 @@
 use crate::time;
+use crate::vga_logger::VgaWriter;
 
 use core::fmt;
 use core::fmt::Write;
 use spin::Mutex;
 
 static LOG_LOCK: Mutex<()> = Mutex::new(());
+static VGA_LOCK: Mutex<VgaWriter> = Mutex::new(VgaWriter::new());
 
-pub fn write_console(s: impl AsRef<str>) {
+pub fn write_console(s: &str) {
```

Comment:
Why change this from AsRef?

Reply:
accidentally left this in here, meant to change it back to `impl AsRef`